packages/checkout/widgets-lib/src/widgets/sale/functions/sortAndDeduplicateCurrencies.test.ts

```diff
@@ -0,0 +1,197 @@
+import { SaleWidgetCurrency, SaleWidgetCurrencyType } from '../types';
+import { sortAndDeduplicateCurrencies } from './sortAndDeduplicateCurrencies';
+
+describe('sortAndDeduplicateCurrencies', () => {
+  it('sortAndDeduplicateCurrencies', () => {
+    const allCurrencies: SaleWidgetCurrency[] = [
+      {
```

Comment:
@mimi-imtbl not a blocker, the input array is already sorted, can you scramble the order to make sure the sort function is actually doing its job?